src/nvim/strings.c

```diff
@@ -832,15 +840,22 @@ int vim_vsnprintf_typval(
       // parse flags
       while (true) {
         switch (*p) {
-          case '0': zero_padding = 1; p++; continue;
-          case '-': justify_left = 1; p++; continue;
-                    // if both '0' and '-' flags appear, '0' should be ignored
-          case '+': force_sign = 1; space_for_positive = 0; p++; continue;
-          case ' ': force_sign = 1; p++; continue;
-                    // if both ' ' and '+' flags appear, ' ' should be ignored
-          case '#': alternate_form = 1; p++; continue;
-          case '\'': p++; continue;
-          default: break;
+        case '0':
+          zero_padding = 1; p++; continue;
+        case '-':
+          justify_left = 1; p++; continue;
+        // if both '0' and '-' flags appear, '0' should be ignored
+        case '+':
+          force_sign = 1; space_for_positive = 0; p++; continue;
+        case ' ':
+          force_sign = 1; p++; continue;
+        // if both ' ' and '+' flags appear, ' ' should be ignored
+        case '#':
+          alternate_form = 1; p++; continue;
```

Comment:
I wonder why it doesn't put `continue` on a newline

Reply:
Hmm, good question. I did a quick look and I don't think there's an option for that yet. I'll add it to my TODO list :)